test/performance/odp_ipsec.c

```diff
@@ -809,6 +809,16 @@ run_measure_one_config(ipsec_args_t *cargs,
 		return -1;
 	}
 
+	if (cargs->schedule && capa.queue_type_sched == 0) {
```

Comment:
capa.queue_type_sched is odp_boolean_t so I think '!capa.queue_type_sched' (or 'capa.queue_type_sched == false') would be better.

Reply:
Initially I kept it  'capa.queue_type_sched == false' but it gave checkpatch error so kept it this way. I will update this to '!capa.queue_type_sched'.